### PR TITLE
Bump dev

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Fixes the dev release and bumps the rizin & translation version

Required for https://github.com/rizinorg/cutter/pull/3191

### CODE REVIEW

File: src/CutterApplication.cpp [hunk 1]
```diff
@@ -33,9 +33,9 @@
 // has RZ_GITTAP defined and uses it in rz_core_version().
 // After that, RZ_GITTAP is not defined anymore and RZ_VERSION is used.
 #ifdef RZ_GITTAP
-#define CUTTER_COMPILE_TIME_RZ_VERSION "" RZ_GITTAP
+#    define CUTTER_COMPILE_TIME_RZ_VERSION "" RZ_GITTAP
```
Comment (XVilka) [inline src/CutterApplication.cpp:36]: Unnecessary change

File: src/CutterApplication.cpp [hunk 2]
```diff
@@ -174,16 +175,16 @@ CutterApplication::CutterApplication(int &argc, char **argv) : QApplication(argc
         rzprefix.cd("Resources"); // Contents/Resources/
 
         auto sleighHome = rzprefix;
-        sleighHome.cd(
-                "lib/rizin/plugins/rz_ghidra_sleigh"); // Contents/Resources/lib/rizin/plugins/rz_ghidra_sleigh
+        // Contents/Resources/lib/rizin/plugins/rz_ghidra_sleigh
+        sleighHome.cd("lib/rizin/plugins/rz_ghidra_sleigh");
```
Comment (XVilka) [inline src/CutterApplication.cpp:179]: All three are the same, should it become a string constant instead?

File: src/translations
Comment (XVilka) [inline src/translations:1]: Please update to the latest cutter-translations, there were recent updates

**Review by XVilka — CHANGES_REQUESTED** [review]

File: src/CutterApplication.cpp (hunk 1 above)
Comment (wargio) [reply src/CutterApplication.cpp:36]: clang-format

File: src/CutterApplication.cpp (hunk 2 above)
Comment (wargio) [reply src/CutterApplication.cpp:179]: i know, but i prefer to keep this as is, in case in the future this changes again.

**Review by XVilka — APPROVED** [review]
Not happy about hardcoded paths, apart from that LGTM
